src/openforms/js/components/admin/forms/VariableSelection.stories.js

```diff
@@ -36,6 +37,20 @@ export default {
         },
 
         formDefinition: 'foo',
+        name: 'foo',
+        slug: '',
+        url: '',
+        _generatedId: '',
+        isNew: true,
+        validationErrors: [],
+      },
+      {
+        configuration: {
+          display: 'form',
+        },
```

Comment:
this looks a bit out of place, but I could be wrong about it!

Reply:
Hmm yeah I think that that can be removed.. Will test it 👍

Reply:
Yeah seems to work fine without it.. Removing it